File: switch_or_match_statement/Exercise_2/switch.cs
Console.Write("Enter a grade: ");
string grade = Console.ReadLine()!;
switch (grade)
{
    case "A":
        Console.Write("4.0");
    break;
    case "B":
        Console.Write("3.0");
    break;
    case "C":
        Console.Write("2.0");
    break;
    case "D":
        Console.Write("1.0");
    break;
    case "F":
        Console.Write("0.0");
        break;
    default:
        Console.Write("You entered an invalid grade");
        break;
}

File: switch_or_match_statement/Exercise_2/switch.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Enter a grade: ", (grade) =>{
switch (grade.trim()){
    case "A":
        console.log("4.0")
    break;
    case "B":
        console.log("3.0")
    break;
    case "C":
    console.log("2.0")
    break;
    case "D":
        console.log("1.0")
    break;
    case "F":
        console.log("0.0")
        break;
    default:
        console.log("You entered an invalid grade")
}
rl.close();
});